### ISSUE TITLE
Suggest screen reader users migrate from `cmd prompt` -> `pwsh`

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#174365]
Shell integration offers many improvements to the terminal experience and makes it more accessible via commands like `Run Recent Command`. 

@jooyoungseo reported that he and many others use `cmd prompt` which does not have shell integration.

Should we include a hint in the help menu that the experience could be better when we detect a screen reader & `cmd prompt`?

**Comment by jooyoungseo** [comment]
I think this will improve the discoverability! Voting for this idea.
